commit 892592da8e560ca4081dd589415e782b4887fe22
Author: Chris Burroughs <chris.burroughs@joyent.com>
Date:   2019-07-31T10:38:19-04:00 (10 weeks ago)
    
    TRITON-1851 update nat base image to triton-origin-x86_64-18.4.0
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/Makefile b/Makefile
index 569f811..0add8f4 100644
--- a/Makefile
+++ b/Makefile
@@ -19,7 +19,7 @@ TOP ?= $(error Unable to access eng.git submodule Makefiles.)
 RELEASE_TARBALL:=$(NAME)-pkg-$(STAMP).tar.gz
 RELSTAGEDIR:=/tmp/$(NAME)-$(STAMP)
 
-BASE_IMAGE_UUID = de411e86-548d-11e4-a4b7-3bb60478632a
+BASE_IMAGE_UUID = a9368831-958e-432d-a031-f8ce6768d190
 BUILDIMAGE_NAME = $(NAME)
 BUILDIMAGE_DESC	= SmartDataCenter per-user NAT zone
 
